### PR TITLE
Update dev environment to F33

### LINKED ISSUE #131: Update Vagrant to F33
Vagrant is currently running F30, which is no longer available from official vagrant source. It is needed to update it to newer version to have ready to use development environment.

## Acceptance Criteria
* Tests are running in F33
* Documentation is updated
* Minikube is running in F33

## Definition of Done
* Documentation is updated
* Tests are passing
* Deployment is working in Vagrant

### PR DESCRIPTION
**Pull request by Zlopez**
Fixes #131

## Proposed Changes

* Update vagrant to F33
* Update MOTD for Vagrant
* Use Fedora bundled packages when available

## Verification Steps

* Run `molecule test -s test-local` in Vagrant
* Deploy operator in minikube inside Vagrant

### CODE REVIEW

File: ci/roles/openshift-osdk/tasks/operator-sdk.yml [hunk 1]
```diff
@@ -2,27 +2,22 @@
 - name: Install Operator SDK system dependencies
   dnf:
     name: [
-            golang,
+            ansible,
+            ansible-lint,
             docker,
+            kubernetes,
```
Comment (odra) [inline ci/roles/openshift-osdk/tasks/operator-sdk.yml:8]: Shouldn't it use the fixed version of "11.0.0"?

File: ci/roles/openshift-osdk/tasks/operator-sdk.yml [hunk 2]
```diff
@@ -74,40 +57,65 @@
...
+#
+#- name: Get minkube config
+#  command: "minikube kubectl config view"
+#  register: output
+#
+#- name: Copy config content
+#  copy:
+#    content: "{{ output.stdout }}"
+#    dest: /.kube/config
+#  become: yes
```
Comment (odra) [inline ci/roles/openshift-osdk/tasks/operator-sdk.yml:91]: if this section is commented, can it be deleted?

Comment (Zlopez) [reply ci/roles/openshift-osdk/tasks/operator-sdk.yml:91]: Yes, the minikube and the tests are working without it, I was just testing it.

File: ci/roles/openshift-osdk/files/motd [hunk 3]
```diff
@@ -7,39 +7,44 @@ Here are some tips:
...
 * To deploy and run the mbox operator in the local kubernetes cluster:
+  # First you need to start the minikube cluster and enable the ingress addon
+  $ minikube start
+  $ minikube addons enable ingress
+
   # Build the image and upload it to quay
   # This needs to be done every time you made a change to the operator code
-  # You may also need to update the image name in mbox-operator/deploy/operator.yaml
+  # You may also need to update the IMG variable in mbox-operator/Makefile
   $ operator-sdk build quay.io/username/mbbox-operator:tag
```
Comment (odra) [inline ci/roles/openshift-osdk/files/motd:22]: the `build` subcommand in operator-sdk is gone, the text should indicate to use a container build tool such as docker: `docker build -t quay.io/username/mbbox-operator:tag .`

Comment (Zlopez) [reply ci/roles/openshift-osdk/files/motd:22]: Didn't actually tried that, good catch

**Comment by odra** [comment]
LGTM, just a minor comment on the MOTD message.

**Comment by Zlopez** [comment]
Fixed the MOTD

**Review by odra — APPROVED** [review]

**Review by odra — APPROVED** [review]

**Pull request merged**